tests/functional/Hydro/Noh/Noh-planar-1d.py

```diff
@@ -177,6 +179,7 @@
             dataDirBase = "dumps-planar-Noh",
             restartBaseName = "Noh-planar-1d",
             restartFileConstructor = SiloFileIO,
+            restartFileCount = mpi.procs,
```

Comment:
As per my comment above setting this to `None` stops us from having to define the default behavior again, unless we want to explicitly enforce some other value here.

Reply:
I'm good with these two changes (SPIOFileCount and None as default).  None is consistent with how we're defaulting other similar parameters here.